dts/dts.cmake

```diff
@@ -16,25 +16,30 @@ message(STATUS "Generating zephyr/include/generated/generated_dts_board.h")
 
 if(CONFIG_HAS_DTS)
 
-  set(DTC_INCLUDE_FLAG_FOR_SOURCE
-    -include ${DTS_SOURCE}
-    )
-
-  if(NOT DTC_OVERLAY_FILE)
-    # overlay file is not set, so try to use overlay dir instead.
-
-    set_ifndef(DTC_OVERLAY_DIR ${APPLICATION_SOURCE_DIR})
+  set_ifndef(BOARD_OVERLAY_DIR ${APPLICATION_SOURCE_DIR})
 
-    set(DTC_OVERLAY_FILE ${DTC_OVERLAY_DIR}/${BOARD_FAMILY}.overlay)
+  set(BOARD_OVERLAY_FILE ${DTC_OVERLAY_DIR}/${BOARD_FAMILY}.overlay)
```

Comment:
DTC_OVERLAY_DIR is used without being defined now.

Did you mean BOARD_OVERLAY_DIR instead of DTC_OVERLAY_DIR? If so, why not just use APPLICATION_SOURCE_DIR? The commit log says your intent is to only search in `<appdir>/<board>.overlay`; why the extra variable?

Reply:
Oh, my mistake. I will fix this